### DISCUSSION TITLE
Add filter to Collections and Data Model pages

### DISCUSSION THREAD

**Discussion by bastianhilton** [directus/directus#20657] [Feature Requests]
### Summary

Adding a filter to to properly sort and/or remove certain collections from view would make it easier to navigate a long list of collections in either view. 

### Motivation

Adding a filter to to properly sort and/or remove certain collections from view would make it easier to navigate a long list of collections in either view. 

### Detailed Design

Adding a filter to to properly sort and/or remove certain collections from view would make it easier to navigate a long list of collections in either view. 

### Must Have:
- A  A quick filter button on the Collections view and Data Model page
- B

### Should Have:
- C

### Could Have:
- D

### Won't Have:
- E

### Drawbacks

There aren't any drawback to the filter button as its just an addition and requires little maintenance. Filters are already featured within each collection and gallery views.

### Alternatives

None

### Adoption Strategy

This isn't a breaking change and is purely for the CMS views in Collections and Data Model pages. IT won't change the overall workflow of the system nor would it break any APIs

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
